src/Public.Api/RoadSegment/RoadSegmentController-ChangeAttributes.cs

```diff
@@ -0,0 +1,86 @@
+using Swashbuckle.AspNetCore.Annotations;
+using Swashbuckle.AspNetCore.Filters;
+using ProblemDetails = Be.Vlaanderen.Basisregisters.BasicApiProblem.ProblemDetails;
+
+public partial class RoadSegmentController
+{
+    private const string ChangeAttributesRoute = "wegsegmenten/acties/wijzigen/attributen";
+
+    /// <summary>
+    ///     Attribuutwaarde van status, toegangsbeperking, wegklasse, wegbeheerder en wegcategorie van wegsegmenten wijzigen.
+    /// </summary>
+    /// <param name="request"></param>
+    /// <param name="actionContextAccessor"></param>
+    /// <param name="problemDetailsHelper"></param>
+    /// <param name="featureToggle"></param>
+    /// <param name="cancellationToken"></param>
+    /// <response code="202">Als het wegsegment gevonden is.</response>
+    /// <response code="400">Als uw verzoek foutieve data bevat.</response>
+    /// <response code="404">Als het wegsegment niet gevonden kan worden.</response>
+    /// <response code="412">Als de If-Match header niet overeenkomt met de laatste ETag.</response>
+    /// <response code="429">Als het aantal requests per seconde de limiet overschreven heeft.</response>
+    /// <response code="500">Als er een interne fout is opgetreden.</response>
+    [HttpPost(ChangeAttributesRoute, Name = nameof(ChangeAttributes))]
+    [ProducesResponseType(StatusCodes.Status202Accepted)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status412PreconditionFailed)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status429TooManyRequests)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+    [SwaggerResponseHeader(StatusCodes.Status202Accepted, "ETag", "string", "De ETag van de response.")]
+    [SwaggerResponseHeader(StatusCodes.Status202Accepted, "x-correlation-id", "string", "Correlatie identificator van de response.")]
+    [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status404NotFound, typeof(RoadSegmentNotFoundResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status412PreconditionFailed, typeof(PreconditionFailedResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status429TooManyRequests, typeof(TooManyRequestsResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+    [SwaggerRequestExample(typeof(ChangeRoadSegmentAttributesParameters), typeof(ChangeRoadSegmentAttributesParametersExamples))]
+    [SwaggerOperation(OperationId = nameof(ChangeAttributes), Description = "Attributen wijzigen van een wegsegment: status, toegangsbeperking, wegklasse, wegbeheerder en wegcategorie.")]
+    public async Task<IActionResult> ChangeAttributes(
```

Comment:
Name should match toggle without `Toggle` to work correctly by convention.
So this should be `RoadSegmentChangeAttributes`.

Reply:
and check other actions as well

Reply:
The class names for FeatureToggles are with Toggle in every other part of the API also? In configuration this needs to be without Toggle indeed, which it is. Are we missing something?

Reply:
Normalized actions and toggles and route definitions